src/test/java/com/synopsys/integration/detect/battery/docker/GradleNativeInspectorTests.java

```diff
@@ -0,0 +1,52 @@
+package com.synopsys.integration.detect.battery.docker;
+
+import com.synopsys.integration.detect.battery.docker.provider.BuildDockerImageProvider;
+import com.synopsys.integration.detect.battery.docker.util.DetectCommandBuilder;
+import com.synopsys.integration.detect.battery.docker.util.DetectDockerTestRunner;
+import com.synopsys.integration.detect.battery.docker.util.DockerAssertions;
+import com.synopsys.integration.detect.configuration.DetectProperties;
+import com.synopsys.integration.detector.base.DetectorType;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+
+@Tag("integration")
+public class GradleNativeInspectorTests {
+    
+    @Test
+    void gradleInspector_7_6() throws IOException, InterruptedException {
+        try (DetectDockerTestRunner test = new DetectDockerTestRunner("detect-gradle-native-inspector", "gradle-simple:1.0.0")) {
```

Comment:
Should we be more specific with the 2nd argument like we are for Gradle 8.2? Something like `gradle-simple-7-6:1.0.0`?

Reply:
The current gradle-simple is tested for support up to 7.2 already. Hence, it did not make sense to create a new file for another minor version under 7.x.